src/bundle/Editor.ts

```diff
@@ -123,6 +123,8 @@ export type MarkupConfig = {
      * See more https://codemirror.net/docs/ref/#state.EditorState.languageDataAt
      */
     languageData?: YfmLangOptions['languageData'];
+    /** Autocompletion config. Can be used to style tooltip */
+    autocompletion?: CreateCodemirrorParams['autocompletion'];
```

Comment:
Plz add a few more comments to describe that this is the configuration for `@codemirror/autocomplete` extension. And add link to reference manual: https://codemirror.net/docs/ref/#autocomplete.autocompletion%5Econfig